docs/dx-eq.rst

```diff
@@ -86,12 +86,20 @@ Schema definition
         .. code-block:: javascript
 
             [{
-                "value": "Joe Bloggs",
+                "value": "Joe Bloggs"
                 "block_id": "household-composition",
-                "answer_id": "household-full-name",
+                "answer_id": "household-first-name",
                 "group_id": "multiple-questions-group",
                 "group_instance": 0,
                 "answer_instance": 0
+            },
+            {
```

Comment:
Is it worth adding a comment here to define why there are multiple values over having answer_instances i.e. because it's a checkbox type answer